circle.yml

```diff
@@ -1,15 +1,13 @@
-dependencies:
-  pre:
-    - rm -r ~/virtualenvs
-
 machine:
   python:
-    version: 2.7
+    version: '2.7.10'
+
+dependencies:
+  override:
+    - pip install -U pip
+    - pip install -U tox tox-pyenv
+    - pyenv local 2.7.9 3.4.3 3.5.0
```

Comment:
@jarosser06 @martinb3 looks like if you use the trusty image you can have the latest: https://circleci.com/docs/trusty-image/